src/yang/ietf-bgp-policy.yang

```diff
@@ -559,6 +596,49 @@ module ietf-bgp-policy {
           }
         }
       }
+
+      container set-large-community {
+        description
+          "Action to set the large community attributes of the
+           route, along with options to modify how the community is
+           modified. Extended communities may be set using an inline
```

Comment:
Should this say Extended or Large communities?

Reply:
Large.  This block was created by copy, paste, and change and I missed the change.